docs/extensibility/display-section.md

```diff
@@ -311,6 +312,43 @@ ControlledBy widgets render the kind and the name with a link to the resources t
 
 <img src="./assets/display-widgets/ControlledBy--kindOnly.png" alt="Example of a ControlledBy widget without name link" width="40%" style="border: 1px solid #D2D5D9">
 
+### ExternalLink
+
+ExternalLink widgets render the link to external page.
+
+#### Widget-specific parameters
+
+- **linkFormula** - an optional jsonata function to generate custom link.
+- **textFormula** - an optional string or item which will be displayed as link.
+
+#### Examples
+
+##### linkFormula and textFormula usage
+
+```json
```

Comment:
Can you change the code examples to yaml?
See https://github.com/kyma-project/busola/issues/1873 for reference